CompanyNormalization.md

```diff
@@ -19,6 +19,8 @@ The following parameters may be used in constructing a request to the service:
 -        state (optional) : The state component of the provided company's location.
 -        city (optional) : The city component of the provided company's location.
 -        max_results (optional) : The maximum number of results to be returned, between 1 and 10 (inclusive). Defaults to 3.
+-        use_query_classifier (optional) : Filters out results for bad queries. In the case where a nonsense query is provided to the service, no results will will be returned in the response. This value is defualted to true, in order to turn it off users must provide the param with a value of false.
```

Comment:
there is a redundant "will" in the second line. And I think you need a comma between "turn it off" and "users...."